sites.json

```diff
@@ -3791,6 +3791,17 @@
             "lovefilm.com"
         ]
     },
+    
+    {
+		"name": "Lucidchart/Lucidpress",
+		"url": "https://www.lucidchart.com/users/login",
```

Comment:
Is this the most direct link possible? Based on the notes, it sounds like there would be a more direct link available.